app/views/posts/index.html.erb

```diff
@@ -1 +1,13 @@
-<h1>See Blog Posts </h1>
\ No newline at end of file
+<%= render "users/user", user: @user%>
+<div class="container">
+  <% @posts.each_with_index do |post, index| %>
+    <%= render partial: "post" , locals: { post: post, index: index } %>
+      <ul>
+        <% post.comments.each do |comment| %>
+          <li>
+             <%= render "comments/comment", comment: comment %>
+          </li>
```

Comment:
- [ ] I have to see only the 5 most recent comments on this page, not all comments. Kindly fix it.

Reply:
thanks for the review.i have made necessary changes